module/Photo/view/photo/album/index.phtml

```diff
@@ -63,7 +63,9 @@ $this->headTitle($this->translate('Photos'));
         </ol>
     </div>
 </section>
-
+<div class="alert alert-danger" role="alert">
```

Comment:
Consider moving the alert inside the row specified on line 71. Moving it inside this DIV will automatically center align the message and add some spacing after the breadcrumb.

```
<div class="row">
   <div class="col-lg-12">
      <div class="alert alert-danger">...</div>
   </div>
   ...
</div>
```

Reply:
Like this?
![Screenshot 2019-09-30 at 22 06 45](https://user-images.githubusercontent.com/7968933/65912603-c0dd2f80-e3ce-11e9-8c47-f21317908239.png)

Reply:
Yes